### ISSUE TITLE
Additional basic visualization

### ISSUE THREAD

**Issue by ConnectedSystems** [SALib/SALib#257]
I've added some convenience plotting methods to SALib and would appreciate any suggestions/feedback before I submit a PR.

All functions/methods return matplotlib axes objects.

Changes are currently sitting in my [basic-viz branch](https://github.com/ConnectedSystems/SALib/tree/basic-viz)

Example usage code and resulting figures provided below. Should be possible to start playing around after a copy/paste.

A possible source of brittleness: currently relying on confidence bound values to be provided in columns with `_conf` in the header.

Looking forward to hearing from you all.

```python
import numpy as np
from SALib.sample import saltelli
from SALib.analyze import sobol

from SALib.sample.fast_sampler import sample as fast_sampler
from SALib.analyze.fast import analyze as fast_analyzer

from SALib.test_functions import Ishigami

# Set figure attributes
import matplotlib
matplotlib.rc('figure', figsize=(6, 6))


problem = {
  'num_vars': 3,
  'names': ['x1', 'x2', 'x3'],
  'bounds': [[-np.pi, np.pi]]*3
}

samples = fast_sampler(problem, 2000, seed=101)
Y = Ishigami.evaluate(samples)
Si = fast_analyzer(problem, Y, print_to_console=False)

# New convenience plotting method
# See Figure 1 below
Si.plot()

# Also possible:
from SALib.plotting.bar import plot as barplot

samples = saltelli.sample(problem, 2000, seed=101, calc_second_order=False)
Y = Ishigami.evaluate(samples)
Si_saltelli = sobol.analyze(problem, Y, print_to_console=False, calc_second_order=False)
Saltelli_T, Saltelli_1st = Si_saltelli.to_df()

# More direct control by manually specifying figure attributes
# See Figure 2 below
fig, (ax1, ax2) = plt.subplots(1,2, figsize=(16,6))

ax1 = barplot(Saltelli_T, ax=ax1)
ax2 = barplot(Saltelli_1st, ax=ax2)
```

**Figure 1**
![image](https://user-images.githubusercontent.com/4075136/62944129-63d9ea00-be1f-11e9-86a2-c1aa7b908a74.png)

**Figure 2**
![image](https://user-images.githubusercontent.com/4075136/62944138-69373480-be1f-11e9-9d6d-42eacb5c91b5.png)

**Comment by jdherman** [comment]
Hi @ConnectedSystems , good idea to include this. Thanks! In the past I always punted on any visualization stuff, but I can definitely see the value in a simple call to Si.plot(). From the code it looks like this will work for all of the methods, too - but I might be forgetting something.

Anyway I would definitely encourage you to submit a PR. It's really useful and could be the foundation for more plot types in the future.

**Comment by jdherman** [comment]
closed by #259
